palindrom-client.html

```diff
@@ -246,18 +241,8 @@
                     operation = patchesAndResults.patches[operationNo];
                     const result = patchesAndResults.results[operationNo];
                     if (operation.path == "") {
-                        //in case of root path, Polymer replaces the original obj, which detaches it from Palindrom
-                        const newSequence = { patches: [], results: [] };
-                        for (const prop in operation.value) {
-                            if (operation.value.hasOwnProperty(prop)) {
-                                newSequence.patches.push({
-                                    op: "add",
-                                    path: `/${prop}`,
-                                    value: operation.value[prop]
-                                });
-                            }
-                        }
-                        this.notifyTemplateDomBind(tree, newSequence, templateDomBind, polymerPathPrefix);
+                        // Replate the original object with the new one.
+                        templateDomBind.set('model', this.obj);
                         continue;
```

Comment:
If whole object is replaced, do we need to bother about other props? I think `return;` here would do.

Reply:
I think we should bother, theoretically we can get such patch:
```
[
    {op: 'replace', path: '', value: {foo: 'bar'}},
    {op: 'add', path: '/bar', value: 'baz'}
]
```
Then the end result of object bound and notified to Polymer should be:
```
{
    foo: 'bar',
    bar: 'baz'
}
```
not just `{foo: 'bar'}`

Reply:
But we're not giving the reference (not the value) to the whole object. Anything inside it will be handed over to domBind. We don't need to care about props with in since we're umbrella-setting the whole object. I hope I'm getting this right.

Reply:
But still, next operation object in the path, may change something in this object.

Reply:
I don't think it can change. By the time we're here all the operations in the patch are applied to the object. See

https://github.com/Palindrom/Palindrom/blob/master/src/palindrom.js#L814-L843

Reply:
Ahhh.. now I get it, you're right, thanks 
:)